### PR TITLE
Always set input_filename

### PR DESCRIPTION
**Pull request by alkino**

### CODE REVIEW

File: src/mod2c_core/modl.c [hunk 1]
```diff
@@ -87,8 +87,10 @@ static void openfiles(char* given_filename, char* output_dir) {
   modprefix = strdup (given_filename);                      // we want to keep original string to open input file
   char* first_ext_char = strrchr(modprefix, '.');           // find last '.' that delimit file name from extension
 
+  Sprintf(input_filename, "%s", given_filename);
```
Comment (pramodk) [inline src/mod2c_core/modl.c:90]: given_filenae was not null terminated?

**Review by pramodk — APPROVED** [review]

**Review by pramodk — APPROVED** [review]

File: src/mod2c_core/modl.c (hunk 1 above)
Comment (alkino) [reply src/mod2c_core/modl.c:90]: ?

Comment (alkino) [reply src/mod2c_core/modl.c:90]: hum, no, we use input_filename line 116 without being initialized if the file exists

**Comment by alkino** [comment]
Wait BlueBrain/Coreneuron#409 before merging

File: src/mod2c_core/modl.c (hunk 1 above)
Comment (pramodk) [reply src/mod2c_core/modl.c:90]: Oh I see!

**Comment by pramodk** [comment]
Simulation stack finishes.

**Pull request merged**